device-services/device-random/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I think this file should be move to the top of the repo (since we don't have one) and not repeated in the examples.

Reply:
Agree. Moved to the top of the repo..